testing_and_setup/compass/ocean/global_ocean/scripts/create_E3SM_coupling_files.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python
+"""
+This script creates coupling files needed to run an MPAS-Ocean and
+MPAS-seaice within E3SM.
+
+Load the lastest e3sm-unified conda package.
+
+"""
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+import os
+import shutil
+import subprocess
+from six.moves.configparser import ConfigParser
```

Comment:
Again, could use the python 3 version of the config parser if we're dropping python 2.7 support.

Reply:
Agreed.